lib/Doctrine/DBAL/Platforms/OraclePlatform.php

```diff
@@ -418,9 +418,32 @@ protected function _getCreateTableSQL($table, array $columns, array $options = a
      */
     public function getListTableIndexesSQL($table, $currentDatabase = null)
     {
-        $table = $this->normalizeIdentifier($table);
-        $table = $this->quoteStringLiteral($table->getName());
+        if (null !== $table) {
+            $table = $this->normalizeIdentifier($table);
```

Comment:
Don't overwite the `$table` variable - create new variables every time the semantic meaning changes